common/djangoapps/pipeline_mako/templates/static_content.html

```diff
@@ -19,6 +20,11 @@
 from lang_pref.api import released_languages
 %>
 
+<%def name="marketing_link(name)"><%
+    link = marketing_link(name)
+    return "/" if link == "#" else link
```

Comment:
If the marketing link returns `#` I'm printing a `/` so the link is valid. 

I originally tried changing what the marketing link spits out (`/` instead of `#`) but things break majorly and it goes into several other things. This adjustment came as a recommendation from @saleem-latif who was familiar with the marketing link stuff.